buckets/storage.py

```diff
@@ -18,6 +19,7 @@ def __init__(self):
         self.secret_key = settings.AWS['SECRET_KEY']
         self.region = settings.AWS['REGION']
         self.bucket_name = settings.AWS['BUCKET']
+        self.max_size = settings.AWS.get('MAX_FILE_SIZE', MAX_FILE_SIZE)
```

Comment:
I would personally expect that there would be no default filesize to an S3 library.  I'd recommend defaulting to no filesize limit, only limiting if someone opts-in to this feature.